app/experimenter/experiments/migrations/0170_restore_feature_configs.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.2 on 2021-05-10 21:34
+
+from django.db import migrations
+
+
+def restore_feature_configs(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+    NimbusFeatureConfig = apps.get_model("experiments", "NimbusFeatureConfig")
+    for experiment in NimbusExperiment.objects.all():
+        feature_slug = (
+            experiment.changes.all()
+            .order_by("changed_on")
```

Comment:
TL;DR: I think this should be `.order_by("-changed_on")` in order to catch the latest change and not the earliest change of feature config.

I was able to see my hunch happen:

1. `git co 91cd41d4`
2. `git co HEAD^` (i.e. the commit before the bad migration)
3. Create a draft experiment, change the feature config a few times
4. `git co main`
5. Run migrations
6. `git co upstream/5216`
7. Run migrations
8. The experiment was reset to the *first* feature config and not the *final* feature config before the migration